### ISSUE TITLE
Support ImportQualifiedPost

### ISSUE THREAD

**Issue by cblp** [jaspervdj/stylish-haskell#284]
Starting from GHC 8.10, with this extension enabled, one can write:

```haskell
import A qualified
import B
import C
```

Now we can save 10 spaces on non-qualified imports!

```haskell
module Bad where

import Control.Applicative ((<$>))
import System.Directory    (doesFileExist)

import Data.Map            (Map, keys, (!))
import Data.Map            qualified as M
```

**Comment by EncodePanda** [comment]
Yes, we need that as well at work :)
Stylish-haskell must first switch to a new parser library (from haskell-src-ext to ghc-lib-parser) in order for us to provide this feature.

**Comment by cblp** [comment]
HLint can tell a success story about such transition.

**Comment by EncodePanda** [comment]
HLint is a good inspiration :)

**Comment by jaspervdj** [comment]
This should be fixed by #307.

**Comment by cblp** [comment]
cool!
